.github/workflows/maven.yml

```diff
@@ -0,0 +1,139 @@
+#
+# WSO2 LLC. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Build Open Banking Repository
+
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  pull_request:
+    branches:
+      - 'master'
+    types:
+      - labeled
+
+jobs:
+
+  Build-repo-check:
+
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'Ready-to-Build') }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - uses: actions/checkout@v3
+      - name: Set up Node 16
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Cache local Maven repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - name: Install DNS-Utils
+        run: sudo apt-get install dnsutils
+      - name: Add hosts to /etc/hosts
+        run: |
+          IP_Address=$(dig +short internal-Private-mvn-ent-wso2-com-LB-1306279662.us-east-1.elb.amazonaws.com | tail -n1)
+          sudo echo $IP_Address "maven.enterprise.wso2.com" | sudo tee -a /etc/hosts
+      - name: Install Open VPN
+        run: sudo apt-get install openvpn
+      - name: Setup VPN config
+        run: |
+          echo "${{ secrets.VPN_FILE }}" > .github/workflows/config.ovpn
+      - name: Connect VPN
+        run: sudo openvpn --config ".github/workflows/config.ovpn" --daemon
+      - name: Setup Maven settings.xml
+        uses: whelk-io/maven-settings-xml-action@v11
+        with:
+          repositories: >
+            [
+              {
+                "id": "enterprise-maven",
+                "name": "enterprise-maven-repository",
+                "url": "https://maven.enterprise.wso2.com/repository/wso2-releases/",
+                "releases": {
+                  "enabled": "true",
+                  "updatePolicy": "daily",
+                  "checksumPolicy": "ignore"
+                }
+              }
+            ]
+          plugin_repositories: >
+            [
+              {
+                "id": "enterprise-maven",
+                "name": "enterprise-maven-repository",
+                "url": "https://maven.enterprise.wso2.com/repository/wso2-releases/",
```

Comment:
We can change this URL as well

Reply:
fixed with https://github.com/wso2/financial-open-banking/pull/1/commits/b60de99a54042245844ebf178bae20872ffcd822